### PR TITLE
**Breaking:** Page title redesign.

### PR DESCRIPTION
**Pull request by ImogenF**
# Summary

Page titles redesigned/simplified as per the new specs. 

https://contiamo.atlassian.net/secure/RapidBoard.jspa?rapidView=43&projectKey=UI&modal=detail&selectedIssue=UI-50

Tabs will be updated separately, at which point the `condensedTitle` prop will also be considered.

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ const font = {
   lineHeight: 1.4,
   size: {
     /** 16px */
```
Comment (stereobooster) [inline src/utils/constants.ts]: Please update comment as well

Comment (ImogenF) [reply src/utils/constants.ts]: Good catch!

**Comment by stereobooster** [comment]
New
Screenshot 2019-06-06 at 14 53 45

vs old
Screenshot 2019-06-06 at 14 53 55

There is no color difference in header and body, is it intended?

**Comment by ImogenF** [comment]
@stereobooster According to the specs, doesn't look like there should be a colour difference. The specs all have white backgrounds though, so I could be wrong... 

image

File: src/Page/Page.tsx [hunk 2]
```diff
@@ -17,10 +15,6 @@ export interface BaseProps extends DefaultProps {
   title?: string
   /** Page actions, typically `condensed button` component inside a fragment */
   actions?: React.ReactNode
-  /** Actions position */
-  actionsPosition?: "start" | "main" | "end"
```
Comment (stereobooster) [inline src/Page/Page.tsx]: this is breaking change (if I understand it right), so PR should be marked as **Breaking**

**Comment by TejasQ** [comment]
# [We should probably add back the actions](https://contiamo.slack.com/archives/C0PF194MT/p1559741500014800)

**Pull request merged**